index.js

```diff
@@ -29,6 +30,10 @@ function all (list) {
   let offset = 0
   for (let i = 0; i < list.length; i++) {
     let buf = list[i]
+    if (buf == null) {
+      result[i] = buf
+      continue
+    }
```

Comment:
just add buf !== null to the below if

Reply:
Done. Good to verify that for both this one and the one below, we want to exclude `undefined` entries from the special case (undefined entries would cause a crash--including them is changing `===` to `==`)